cypress/features/regression/flatTabel.feature

```diff
@@ -0,0 +1,89 @@
+Feature: FlatTable component
```

Comment:
typo in file name, should be `flatTable`

Reply:
fixed